priv/static/specs/http_api_spec.yaml

```diff
@@ -55,6 +55,25 @@ paths:
   /contexts:
     delete:
       summary: 'delete all contexts'
+      parameters:
+        - name: version
+          in: query
+          required: false
+          type: string
+          description: 'Version'
+          default: 'v1'
+        - name: imsi
+          in: query
+          required: false
+          type: integer
+          description: 'International Mobile Subscriber Identity'
+          default: '111111111111111'
+        - name: pfcp
```

Comment:
The parameter names makes no sense IMHO. PFCP is a protocol, the entity speaking that protocol is a UPF or in 5G terms "a NF Instance of type `UPF`".

The parameter name should therefor IMHO be `upf-fqdn`.

When a 5G compatible naming is used, then using two arguments, `target-nf-type`, and `target-nf-instance-id` or `target-nf-fqdn` (see TS 29.510, Table 6.2.3.2.3.1-1 for example).

NOTE: In 5G nf-instance-id is a FQDN, but does not need to match the FQDN for instance selection. Therefor nf-fqdn and fn-instance-id could differ.

Reply:
One more thing, the UPF node id is currently missing from the PFCP interaction. This is a oversight that should be fixed soonish (a PR in the UPG project is already pending).